### PR TITLE
Add Information About Content Staging

### LINKED ISSUE #6044: More information about staging functionality
# General issue
Not enough information

https://devdocs.magento.com/guides/v2.3/extension-dev-guide/staging.html

## Description:

This topic provide very high-level information about Staging functionality.   However it does provide any information on whether the listed modules are; and if so how.  Nor does this topic provide a link to corresponding documentation in the User Guide.  As a developer you have to understand how your users are expected or might use the components before you start trying to design or develop a solution using the components.

### PR DESCRIPTION
**Pull request by BarnyShergold**
## Purpose of this pull request

This pull request (PR) fixes #6044 

## Affected DevDocs pages
https://devdocs.magento.com/guides/v2.3/extension-dev-guide/staging.html
https://devdocs.magento.com/guides/v2.4/extension-dev-guide/staging.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/extension-dev-guide/staging.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ ee_only: true
 
 Staging functionality enables you to create scheduled campaigns, manage campaigns on dashboard and manipulate each future store object update. Moreover, you can preview a [storefront](https://glossary.magento.com/storefront) during the future campaign.
 
+{:.bs-callout-info}
+Information on how the Staging functionality works and how to use the various features, can be found on the [Content Staging Guide]({{ site.baseurl }}/user-guide/cms/content-staging.html).
```
Comment (rogyar) [inline src/guides/v2.3/extension-dev-guide/staging.md:10]: Hi @BarnyShergold. Could you double-check that the link goes to an existing page? For some reason, I can't find it. 

Thanks

Comment (BarnyShergold) [reply src/guides/v2.3/extension-dev-guide/staging.md]: Found the problem - right page, wrong domain. It's odd, I'll swear it was working :)

**Review by rogyar — APPROVED** [review]

**Comment by dobooth** [comment]
Is there a reason this needs to be a callout?

**Comment by BarnyShergold** [comment]
@dobooth =- No, just wanted it to standout. Can make it just a normal paragraph if you think it should be.

File: src/guides/v2.3/extension-dev-guide/staging.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ ee_only: true
 
 Staging functionality enables you to create scheduled campaigns, manage campaigns on dashboard and manipulate each future store object update. Moreover, you can preview a [storefront](https://glossary.magento.com/storefront) during the future campaign.
 
+{:.bs-callout-info}
```
Comment (hguthrie) [inline src/guides/v2.3/extension-dev-guide/staging.md:9]:
```suggestion
{:.bs-callout-tip}
```

File: src/guides/v2.3/extension-dev-guide/staging.md [hunk 3]
```diff
@@ -6,6 +6,9 @@ ee_only: true
...
+Information on how the Staging functionality works and how to use the various features, can be found on the [Content Staging Guide](https://docs.magento.com/user-guide/cms/content-staging.html).
```
Comment (hguthrie) [inline src/guides/v2.3/extension-dev-guide/staging.md:10]:
```suggestion
Information on how the Staging functionality works and how to use the various features, can be found in the [Content Staging](https://docs.magento.com/user-guide/cms/content-staging.html) topic of the _Magento User Guide_.
```

**Review by hguthrie — CHANGES_REQUESTED** [review]
If you think this is a good link for this section, then I recommend making it a "tip".
Also, since we're technically jumping guides with that link, it's best to call out which guide using italics.

File: src/guides/v2.3/extension-dev-guide/staging.md (hunk 2 above)
Comment (BarnyShergold) [reply src/guides/v2.3/extension-dev-guide/staging.md:9]: I'm good with that! Thanks for the tip [sic]

File: src/guides/v2.3/extension-dev-guide/staging.md (hunk 3 above)
Comment (BarnyShergold) [reply src/guides/v2.3/extension-dev-guide/staging.md:10]: Good spot

**Review by hguthrie — APPROVED** [review]

**Comment by hguthrie** [comment]
running tests

**Pull request merged**